### PR TITLE
m

### PR DESCRIPTION
**Pull request by loctx-2273**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ class ApplicationController < ActionController::Base
   private
   def set_locale
     I18n.locale = params[:locale] || I18n.default_locale
+    # sfafsd
```
Comment (loctx-2273) [inline app/controllers/application_controller.rb:9]: cat

Comment (loctx-2273) [reply app/controllers/application_controller.rb:9]: sdfsadf

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@ class ApplicationController < ActionController::Base
   private
   def set_locale
```
Comment (loctx-2273) [inline app/controllers/application_controller.rb:7]: sdfsdfa

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -6,6 +6,7 @@ class ApplicationController < ActionController::Base
...
     I18n.locale = params[:locale] || I18n.default_locale
```
Comment (loctx-2273) [inline app/controllers/application_controller.rb:8]: sdffsdfds

Comment (loctx-2273) [reply app/controllers/application_controller.rb:8]: sdfsfd

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (loctx-2273) [reply app/controllers/application_controller.rb:7]: sdfsdfa

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (loctx-2273) [reply app/controllers/application_controller.rb:9]: sadfsdf

File: app/controllers/application_controller.rb (hunk 3 above)
Comment (loctx-2273) [reply app/controllers/application_controller.rb:8]: lonnhe

Comment (loctx-2273) [reply app/controllers/application_controller.rb:8]: fssdfadsf

**Comment by tranloc171197** [comment]
sadfsdfs

**Comment by tranloc171197** [comment]
sadfsdf

**Comment by tranloc171197** [comment]
sdafsdf

**Comment by tranloc171197** [comment]
fdssfda

**Comment by tranloc171197** [comment]
sdaffsdfsda

File: app/controllers/application_controller.rb (hunk 3 above)
Comment (tranloc171197) [reply app/controllers/application_controller.rb:8]: sdfsdfsdffds

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (tranloc171197) [reply app/controllers/application_controller.rb:7]: sdfsdfsdf

**Comment by tranloc171197** [comment]
safdsfd

**Review by tranloc171197 — APPROVED** [review]

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (tranloc171197) [reply app/controllers/application_controller.rb:7]: sdfsdfsd

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (tranloc171197) [reply app/controllers/application_controller.rb:9]: sdfsdfsdfsdf

File: app/controllers/application_controller.rb (hunk 3 above)
Comment (tranloc171197) [reply app/controllers/application_controller.rb:8]: sdfsdfsdf

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (tranloc171197) [reply app/controllers/application_controller.rb:9]: sdfsdfs

File: app/controllers/application_controller.rb (hunk 3 above)
Comment (tranloc171197) [reply app/controllers/application_controller.rb:8]: @oDangDinhLuan @vupx-0677 hehe

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (tranloc171197) [reply app/controllers/application_controller.rb:7]: sadsdafsdaf

File: app/controllers/application_controller.rb (hunk 3 above)
Comment (tranloc171197) [reply app/controllers/application_controller.rb:8]: sdfafdsfdsfds

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (tranloc171197) [reply app/controllers/application_controller.rb:7]: @oDangDinhLuan @vupx-0677 hehe

Comment (tranloc171197) [reply app/controllers/application_controller.rb:7]: @oDangDinhLuan @vupx-0677 hehe

File: app/controllers/application_controller.rb (hunk 3 above)
Comment (tranloc171197) [reply app/controllers/application_controller.rb:8]: @oDangDinhLuan@vupx-0677hehe

Comment (tranloc171197) [reply app/controllers/application_controller.rb:8]: @oDangDinhLuan @vupx-0677 hehe

Comment (tranloc171197) [reply app/controllers/application_controller.rb:8]: @loctx-2273 hehe

Comment (tranloc171197) [reply app/controllers/application_controller.rb:8]: @manhld-0933 rưereưa

Comment (tranloc171197) [reply app/controllers/application_controller.rb:8]: @manhld-0933 afsfsdafsdasdffds

**Comment by tranloc171197** [comment]
sdaffdsfds

**Comment by loctx-2273** [comment]
@sangvv-0944 fsda

**Comment by tranloc171197** [comment]
lgtm

**Comment by tranloc171197** [comment]
LGTM

**Comment by tranloc171197** [comment]
lgtm

**Pull request merged**